css/README.md

```diff
@@ -239,19 +239,126 @@ These rules were adapted from [CSS Guidelines]. This is an example of how declar
-9. Nested pseudo element blocks should come after all property declarations.
-10. Nested pseudo class blocks should come after all pseudo element blocks.
-11. Nested class or attribute blocks should come after all pseudo class blocks.
-12. Nested media queries should come after all combined class or attribute blocks.
+9. Nested pseudo class blocks should come after all property declarations.
+10. Nested class or attribute blocks should come after all pseudo class blocks.
+11. Nested media queries should come after all combined class or attribute blocks.
+12. Nested pseudo element blocks should come after all media queries.
 13. Closing braces should be on a new line.
 14. A blank line should come after each declaration block.
 
+### Order of Nested Selectors
+
+We use a standard order for nested selectors in Sass. By following this order, our code becomes more standardized and easier to understand.
+
+When in doubt, remember that the goal is to first define any **included** styles, followed by **regular** styles, followed by **modifiers**, followed by **children** (whose code would follow this same order).
+
+
+1. #### Included Code
+
+	1. Local `$variables`
+		* Local variables should really only be declared for mixins or functions. Most variables should be kept in the main variables partial, since variables in Sass are global.
+		* However, if you have a set of variables that will only be used in a certain partial, declare them at the top of that file and prefix their names to avoid confusion in the global namespace. e.g., `$m_module_name_height`.
+
+	1. `@extend` statements
+		* Avoid extends in favor of mixins. Extends offer no benefits over mixins, and can cause [specificity problems](http://csswizardry.com/2016/02/mixins-better-for-performance/).
+
+	1. `@include` statements
+		* Knowing right off the bat that this class inherits another whole set of rules from elsewhere is good. Another benefit is that overriding styles for that inherited set of rules becomes much easier.
+
+2. #### Alphabetized Regular Styles
+	1. Adding regular styles after the `@extend` and `@include` statements allows us to override those properties, if needed. Be sure to leave a comment if a declaration's only purpose is to override included styles. 
+    
+	1. Alphabetizing is helpful for a large team because it standardizes location of properties (See "Property Sorting" below).
+
+3. #### Modifiers
+	1. Pseudo-classes (`:hover`), attribute selectors (`&[type="input"]`), and state classes (`&.is-active`)
+		* These directly modify the parent selector so we declare them before other nested selectors.
+        * Most BEM modifier classes don't need to be nested. However, some generic state class names like `is-active` that would otherwise be too non-specific to live in the global namespace, should be nested here.
```

Comment:
It's hard for me to tell, is the indendation here as you intended?

Reply:
Nope, my indenting got messed up. I'll fix it.